babel/messages/frontend.py

```diff
@@ -339,6 +339,8 @@ class ExtractMessages(CommandMixin):
          'Separate multiple patterns with spaces (default ".* ._")'),
         ('header-comment=', None,
          'header comment for the catalog'),
+        ('last-translator=', 'T',
```

Comment:
Maybe skip the shortname?
```suggestion
        ('last-translator=', None,
```

Reply:
I'm good with that. Changed.